cfme/configure/configuration/server_settings.py

```diff
@@ -451,15 +451,279 @@ def _save_action(self, view, updates, reset):
         view.flash.assert_message(flash_message)
 
 
-@navigator.register(ServerInformation)
-class Details(CFMENavigateStep):
+@navigator.register(ServerInformation, 'Details')
+class DetailsServer(CFMENavigateStep):
     VIEW = ServerInformationView
     prerequisite = NavigateToAttribute('appliance.server', 'Details')
 
     def step(self):
         self.prerequisite_view.server.select()
 
 
+# ============================= AUTHENTICATION TAB ===================================
+
+
+class ServerAuthenticationView(ServerView):
+    """ Server Authentication View."""
+    title = Text("//div[@id='settings_authentication']/h3[1]")
+    save_button = Button('Save')
+    reset_button = Button('Reset')
+
+    hours_timeout = BootstrapSelect(id='session_timeout_hours')
+    minutes_timeout = BootstrapSelect(id='session_timeout_mins')
+    authentication_mode = BootstrapSelect(id='authentication_mode')
+
+    @property
+    def is_displayed(self):
+        return (
+            self.authentication_mode.is_displayed and
+            self.title.text == 'Authentication'
+        )
+
+
+class DatabaseAuthenticationView(ServerAuthenticationView):
+    """ Database Authentication View """
+    @property
+    def is_displayed(self):
+        return (
+            self.authentication_mode.is_displayed and
+            self.authentication_mode.selected_option == 'Database'
```

Comment:
This looks like a good candidate for using a conditionalview, since both DatabaseAuthenticationView and LdapAuthenticationView are really dependent on the selection of authentication_mode widget.

Optional for this PR, a `#TODO` might be appropriate here for a later enhancement.

Reply:
Added TODO for this